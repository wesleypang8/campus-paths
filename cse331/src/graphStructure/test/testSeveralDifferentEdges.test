#test several different edge combinations

#create graph
CreateGraph graph1

#add nodes
AddNode graph1 me
AddNode graph1 parent
AddNode graph1 friend

#add edges
AddEdge graph1 parent me p2m
AddEdge graph1 me friend m2f
AddEdge graph1 friend me f2m

#print children/ edges and nodes in graph
ListNodes graph1
ListChildren graph1 parent
ListChildren graph1 me
ListChildren graph1 friend