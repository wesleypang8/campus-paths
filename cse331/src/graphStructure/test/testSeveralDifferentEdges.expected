#test several different edge combinations

#create graph
created graph graph1

#add nodes
added node me to graph1
added node parent to graph1
added node friend to graph1

#add edges
added edge p2m from parent to me in graph1
added edge m2f from me to friend in graph1
added edge f2m from friend to me in graph1

#print children/ edges and nodes in graph
graph1 contains: friend me parent
the children of parent in graph1 are: me(p2m)
the children of me in graph1 are: friend(m2f)
the children of friend in graph1 are: me(f2m)
